config/locales/views/en.yml

```diff
@@ -678,6 +678,10 @@ en:
           hide_content_for_others: hide their works, series, bookmarks, and comments from anyone else
           intro: 'Muting a user will not:'
           prevent_emails: prevent you from receiving comment or subscription emails from this user
+  orphans:
+    orphan_user:
+      orphaning_bylines_only_message_html: Orphaning will automatically remove your personal information from <strong>bylines only.</strong> It will <strong>not</strong> automatically remove your personal information from anywhere else in the work(s). Social media accounts, contact information, email addresses, names, and any other personal information posted in the title, summary, notes, tags, work body, or comment text will <strong>not</strong> be automatically removed. Comments posted by other users will also <strong>not</strong> be affected by Orphaning. If you want information removed from these places, you should edit or delete it prior to Orphaning. <strong>Once you Orphan the work(s), you will no longer be able to delete information in these places yourself.</strong>
```

Comment:
Could you move the period in the first sentence outside the `strong` tag, i.e., `<strong>bylines only</strong>.`?

Reply:
Yes for sure. What do you recommend for work(s)? We could cut it off entirely, end the sentence after "anywhere else.". Or just make works permanently plural? Whatever you think!

Reply:
We can keep `work(s)` as is for now -- it'll be a while before this page is actually translated, and we might want other revisions to the text by then anyway!

Reply:
Awesome thank you, just wanted to make sure. Feedback addressed!